projects/wm-core/src/utils/localForage.ts

```diff
@@ -409,6 +409,15 @@ export async function saveUgcTrack(feature: WmFeature<LineString>): Promise<void
   await handleAsync(storage.setItem(`${featureId}`, feature), 'saveUgcTrack: Failed');
 }
 
+export async function saveUgc(feature: WmFeature<LineString | Point>): Promise<void> {
+  if (!feature || !feature.geometry) return;
```

Comment:
rifattorizzare tutti i controlli null check eseguiti per controllare la consistenza di wmfeature in una unica funzione metterla in utils con un todo spostare in wm-types